masterlist.yaml

```diff
@@ -228,11 +233,16 @@ common:
   - &reqBetterMales
     <<: *reqModX
     subs: [ '[Better males - Beautiful nudes and faces - New hairstyles](http://www.nexusmods.com/skyrim/mods/2488)' ]
-
-  - &reqSkyUI # Useful for when a plugin needs SkyUI, but shouldn't load after it (eg. when it's a master plugin).
+  # Useful for when a plugin needs SkyUI, but shouldn't load after it (eg. when it's a master plugin).
```

Comment:
No, it just bothered me because it blatantly ignored this
> Make as many silent comments as you want within the masterlists - it doesn't hurt (within reason). However, make sure to put them _above_ the line(s) that they refer to.